packages/sdks/snippets/angular-16/src/app/live-preview/live-preview.component.ts

```diff
@@ -0,0 +1,61 @@
+import { CommonModule } from '@angular/common';
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, inject } from '@angular/core';
+import { Router } from '@angular/router';
+import type { BuilderContent } from '@builder.io/sdk-angular';
+import { fetchOneEntry, subscribeToEditor } from '@builder.io/sdk-angular';
+
+@Component({
+  standalone: true,
+  selector: 'app-live-preview',
+  imports: [CommonModule],
+  template: `
+    <div *ngIf="loading">Loading Data...</div>
+
+    <div *ngIf="!loading && content" class="blog-data-preview">
+      <div>Blog Title: {{ content.data?.title }}</div>
+      <div>Authored by: {{ content.data?.['author'] }}</div>
+      <div>Handle: {{ content.data?.['handle'] }}</div>
+    </div>
+  `,
+})
+export class LivePreviewComponent implements OnInit, OnDestroy {
+  content: BuilderContent | undefined = undefined;
+  loading = false;
```

Comment:
lets switch this back to `true` now that we ended up removing the third state:

```suggestion
  loading = true;
```